source/documentation/tools.md

```diff
@@ -22,6 +22,46 @@ Use the buttons shown against each tool to manage your copy of the tool:
 * "Restart" - Often problems with the tool can be solved by restarting the software on the server.
 * "Upgrade" - Another release of the tool is available. Occasionally new versions of R Studio are made available on the Analytical Platform. In this case all users will be given the opportunity to upgrade on the control panel. New versions provide new features and bugfixes to the tool. In addition, there some releases come with improvements to the way RStudio is containerized and integrated with Analytical Platform. You should aim to upgrade when it is offered, although just in case it causes minor incompatibilities with your R code, you should not do it in the days just before you have a critical release of your work. When pressed, the status will change to 'Deploying' and then 'Upgraded'. The __Upgrade__ button will no longer be visible (until another version becomes available).
 
+## Using RStudio
+
+For general guidance in using RStudio, see the [RStudio documentation](https://docs.rstudio.com/)
+
+### RStudio memory issues
+
+It is common to run out of memory in RStudio.
```

Comment:
I wonder if users understand that everything is stored in memory in R, which is what leads to high utilisation. Is there more we can do to encourage good memory management from the outset? I feel like people may think it's an issue with the AP, whereas it's actually an issue with their code.